web_client/models/AnnotationModel.js

```diff
@@ -39,6 +49,55 @@ export default Model.extend({
     },
 
     /**
+     * Fetch a single resource from the server. Triggers g:fetched on success,
+     * or g:error on error.
+     * To ignore the default error handler, pass
+     *     ignoreError: true
+     * in your opts object.
+     */
+    fetch: function (opts) {
```

Comment:
Sadly, the super method doesn't pass through the data parameter.  Instead, it constructs a new set of options to pass to the rest call.